src/main/java/uk/gov/hmcts/ccd/ElasticSearchConfiguration.java

```diff
@@ -15,20 +15,28 @@
 @Configuration
 public class ElasticSearchConfiguration {
 
+    private final ApplicationParams applicationParams;
+
     @Autowired
-    private ApplicationParams applicationParams;
+    public ElasticSearchConfiguration(ApplicationParams applicationParams) {
+        this.applicationParams = applicationParams;
+    }
 
     @Bean
     public JestClient jestClient() {
 
         JestClientFactory factory = new JestClientFactory();
         Gson gson = new GsonBuilder().setFieldNamingPolicy(FieldNamingPolicy.LOWER_CASE_WITH_UNDERSCORES).create();
         factory.setHttpClientConfig(new HttpClientConfig.Builder(applicationParams.getElasticSearchDataHosts())
-            .multiThreaded(true)
-            .maxConnectionIdleTime(15, TimeUnit.SECONDS)
-            .connTimeout(4000)
-            .readTimeout(4000)
-            .gson(gson).build());
+                                        .multiThreaded(true)
+                                        .maxConnectionIdleTime(15, TimeUnit.SECONDS)
+                                        .connTimeout(4000)
+                                        .readTimeout(4000)
```

Comment:
since we're here, can you increase a bit the read timeout? let's do 6 secs?

Reply:
done